Sources/SharedModels/Exercise/Participation/TemplateParticipation.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  TemplateParticipation.swift
+//  
+//
+//  Created by Tarlan Ismayilsoy on 03.05.23.
+//
+
+import Foundation
+
+// Should ideally extend `ProgrammingExerciseStudentParticipation`, but can't since it's a struct. Can be left as it is for now
```

Comment:
Maybe take a look at the StudentParticipation Protocol and solve it similarly. I don't know exactly about the hierarchy here on the serverside, but I think it's solvable in a better way